tests/trino/test-catalogs-secrets-values.yaml

```diff
@@ -0,0 +1,19 @@
+catalogs: {}
+catalogsSecrets:
```

Comment:
What's the advantage of using this over `catalogs`? Please update the PR description with the rationale for the change.